### PR TITLE
FIX: serializes interaction for direct messages

### PR DESCRIPTION
**Pull request by jjaffeux**
Prior to this fix it would cause an error as we need to pass the user to get the title of the channel.

This commit also adds a test for message interaction serializer.

### CODE REVIEW

File: plugins/chat/spec/serializer/chat/message_interaction_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+RSpec.describe Chat::MessageInteractionSerializer do
+  subject(:serializer) do
+    interaction = Chat::CreateMessageInteraction.call(params:, guardian: user.guardian).interaction
```
Comment (Flink) [inline plugins/chat/spec/serializer/chat/message_interaction_spec.rb:4]: Should we create a new fabricator with a `use_service: true` option, like we did for some other models we have in chat?

File: plugins/chat/spec/serializer/chat/message_interaction_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          type: "actions",
+          elements: [
+            { type: "button", text: { type: "plain_text", text: "Like" }, action_id: "like" },
+          ],
+        },
+      ],
+    )
+  end
+  let(:message_id) { message.id }
+  let(:params) { { message_id: message_id, action_id: "like" } }
```
Comment (Flink) [inline plugins/chat/spec/serializer/chat/message_interaction_spec.rb:26]: Maybe a nitpick: we can write `{ message_id:, action_id: "like" }` :slightly_smiling_face:

**Review by Flink — APPROVED** [review]

File: plugins/chat/spec/serializer/chat/message_interaction_spec.rb (hunk 1 above)
Comment (jjaffeux) [reply plugins/chat/spec/serializer/chat/message_interaction_spec.rb:4]: I just didn't bother create the fabricator for now

Comment (Flink) [reply plugins/chat/spec/serializer/chat/message_interaction_spec.rb:4]: Up to you :slightly_smiling_face:

Comment (jjaffeux) [reply plugins/chat/spec/serializer/chat/message_interaction_spec.rb]: I added a basic fabricator for now

**Pull request merged**